cmd/agent/app/server.go

```diff
@@ -152,6 +152,10 @@ func (a *Agent) runAdminServer(o *options.GrpcProxyAgentOptions) error {
 	if o.EnableProfiling {
 		muxHandler.HandleFunc("/debug/pprof", util.RedirectTo("/debug/pprof/"))
 		muxHandler.HandleFunc("/debug/pprof/", pprof.Index)
+		muxHandler.HandleFunc("/debug/pprof/cmdline", pprof.Cmdline)
+		muxHandler.HandleFunc("/debug/pprof/profile", pprof.Profile)
```

Comment:
Is this alone meaningful? (Given that the agent does not integrate with [cpu profiling](https://pkg.go.dev/runtime/pprof#StartCPUProfile))

Reply:
Yeah, `pprof.Profile` does proactive cpu profiling so we don't have to explicitly integrate that elsewhere in the code.

Reply:
ack, thanks. I'll defer to @cheftako to approve.